src/integrationTest/java/uk/gov/hmcts/reform/pip/account/management/controllers/AccountTest.java

```diff
@@ -532,6 +562,41 @@ void testCreateMultipleSuccessUsers() throws Exception {
             objectMapper.readValue(response.getResponse().getContentAsString(),
                                    new TypeReference<>() {});
 
+        assertEquals(1, mappedResponse.get(CreationEnum.CREATED_ACCOUNTS).size(), "1 Users should be created");
+    }
+
+    @Test
+    void testCreateMultipleSuccessUsersWithDifferentEmails() throws Exception {
+
+        when(graphClient.users(any())).thenReturn(userRequestBuilder);
+        when(userRequestBuilder.buildRequest()).thenReturn(userRequest);
+        when(userRequest.get()).thenThrow(graphServiceException);
```

Comment:
do you need to throw here? this looks like youre testing success?

Reply:
My mistake, it should return user object not exception.